iron-location.html

```diff
@@ -319,6 +325,11 @@
       // Need to use a full URL in case the containing page has a base URI.
       var fullNormalizedHref = new URL(
           normalizedHref, window.location.href).href;
+      // If the navigation is to the current page we shouldn't add a history
+      // entry.
+      if (fullNormalizedHref === window.location.href) {
+        return null;
```

Comment:
Just noticing we have a mixed pattern of returning `null` and `undefined` in this method.